<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="2" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-02-17T18:12:52 IST" name="Failed suite [Failed suite [Suitename]]" finished-at="2025-02-17T18:12:52 IST" duration-ms="47">
    <groups>
    </groups>
    <test started-at="2025-02-17T18:12:52 IST" name="Testname(failed)(failed)" finished-at="2025-02-17T18:12:52 IST" duration-ms="47">
      <class name="testing.testAnnotation">
        <test-method is-config="true" signature="bsuite()[pri:0, instance:testing.testAnnotation@a2431d0]" started-at="2025-02-17T18:12:52 IST" name="bsuite" finished-at="2025-02-17T18:12:52 IST" duration-ms="7" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bsuite -->
        <test-method is-config="true" signature="btest()[pri:0, instance:testing.testAnnotation@a2431d0]" started-at="2025-02-17T18:12:52 IST" name="btest" finished-at="2025-02-17T18:12:52 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- btest -->
        <test-method is-config="true" signature="bclass()[pri:0, instance:testing.testAnnotation@a2431d0]" started-at="2025-02-17T18:12:52 IST" name="bclass" finished-at="2025-02-17T18:12:52 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bclass -->
        <test-method signature="TC1()[pri:2, instance:testing.testAnnotation@a2431d0]" started-at="2025-02-17T18:12:52 IST" name="TC1" finished-at="2025-02-17T18:12:52 IST" duration-ms="8" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at testing.testAnnotation.TC1(testAnnotation.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC1 -->
        <test-method is-config="true" signature="aclass()[pri:0, instance:testing.testAnnotation@a2431d0]" started-at="2025-02-17T18:12:52 IST" name="aclass" finished-at="2025-02-17T18:12:52 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aclass -->
        <test-method is-config="true" signature="atest()[pri:0, instance:testing.testAnnotation@a2431d0]" started-at="2025-02-17T18:12:52 IST" name="atest" finished-at="2025-02-17T18:12:52 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- atest -->
        <test-method is-config="true" signature="asuite()[pri:0, instance:testing.testAnnotation@a2431d0]" started-at="2025-02-17T18:12:52 IST" name="asuite" finished-at="2025-02-17T18:12:52 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- asuite -->
      </class> <!-- testing.testAnnotation -->
    </test> <!-- Testname(failed)(failed) -->
  </suite> <!-- Failed suite [Failed suite [Suitename]] -->
</testng-results>
